README

```diff
@@ -44,9 +44,12 @@ SYNOPSIS
             # use -w to prompt for password if w/o SSH key (no echo back)
             $ atnodes hostname '{ps}' -u agentz -w
 
-            # or prompt for password if sudo required...
+            # or prompt for password if login and sudo required...
```

Comment:
Please do not edit README directly since this file was automatically generated from the `lib/SSH/Batch.pm` file instead. Please update the POD source in `lib/SSH/Batch.pm` file instead. (Well, you can leave the `README` file intact and I can take care of the auto generation process on my side) :)